mods/cnc/rules/structures.yaml

```diff
@@ -757,15 +757,15 @@ OBLI:
 		Range: 8c0
 	Bib:
 		HasMinibib: Yes
-	WithChargeAnimation:
+	WithAttackAnimation:
+		AttackSequence: active
+	PreparingAttackSounds:
+		Sounds: obelpowr.aud
 	Armament:
 		Weapon: Laser
 		LocalOffset: 0,-85,1280
-		FireDelay: 0
-	AttackCharge:
-		ChargeAudio: obelpowr.aud
-		ReloadDelay: 40
-		InitialChargeDelay: 50
+		FireDelay: 40
```

Comment:
@GraionDilach pointed out in IRC that this makes the actor ignore the maximum range check, and fire unconditionally on the target.  `AttackCharge` (actually `ChargeFire`) has a specific extra range check that avoids that.